focus-areas/ecosystem/social-currency-metric-system.md

```diff
@@ -1,4 +1,4 @@
-# Social Currency Metric System (SCMS)
+# Ecosystem Value: Social Currency Metric System (SCMS)
```

Comment:
So, I think it helps to use the <code>\`\`\`suggestion</code> fencing like so:

```suggestion
# Social Currency Metric System (SCMS)
```

<small><details>

>     ```suggestion
>     # Social Currency Metric System (SCMS)
>     ```

- It needs to be a seed comment on the line you plan to commit against — ie why I haven't been replying directly to your comments when making `suggestion`s.
- It requires using the UI to make batch or single `suggestion`-commits.
- Not sure how well it pans for multi-line `suggestion`s.
- I use the <kbd>Add single comment</kbd> route, not sure if <kbd>Start a review</kbd> works the same or is reasonably useful.

</details></small>